sdcm/nemesis.py

```diff
@@ -767,7 +767,7 @@ def _choose_file_for_destroy(self, ks_cfs):
 
     def _destroy_data_and_restart_scylla(self):
 
-        ks_cfs = self.cluster.get_non_system_ks_cf_list(db_node=self.target_node)
+        ks_cfs = self.cluster.get_non_system_ks_cf_list(db_node=self.target_node, filter_empty_tables=False)
```

Comment:
The issue is that if it finds an empty table the "destroy_data" won't work.

Reply:
it get a list of all tables, and then randomly select out of it 5 times. so we can move the check of emptiness into destroy_data part ? instead by default do that for all of the tables ?

Reply:
IMHO this would be much better, so we could select 5 non-empty tables?